### ISSUE TITLE
Default Plasma Wallpapers Doesn't Work Anymore

### ISSUE THREAD

**Issue by bentuhana** [luisbocanegra/kde-material-you-colors#91]
**Describe the bug**
KDE Material You Colors does not generate color scheme for default Plasma wallpapers.

**To Reproduce**
1. Run app
2. Change wallpaper to any default Plasma wallpaper
4. It does not work

**Expected behavior**
Color scheme successfully generated.

**Screenshots**
![image](https://user-images.githubusercontent.com/108196023/196529463-dd3b0cdb-dc8b-4be8-a606-db4fb1080291.png)

**Desktop (please complete the following information):**
 - Operating System: Arch Linux
 - KDE Plasma Version: 5.26.1
 - KDE Frameworks Version: 5.99.0
 - Qt Version: 5.15.6
 - Graphics Platform: Xorg
 - Things in your system that might be relevant: *None*

**Additional context**
It's probably because of new dark/light theme wallpapers. With 5.26.0, some default wallpapers now have a dark and light specific version, and instead of they being a file, pictures are in a folder and Plasma still returns a folder path instead of image path.
![image](https://user-images.githubusercontent.com/108196023/196531998-325d470f-c994-476e-a5fc-d42279e7108e.png)

**Comment by luisbocanegra** [comment]
This is the structure of the default wallpapers:
```sh
/usr/share/wallpapers/Next/
├── contents
│   ├── images
│   │   ├── 1024x768.png
...
│   └── images_dark
│       ├── 1024x768.png
...
└── metadata.json

3 directories, 39 files
```
Unless there is another way of querying the current image, I think this could be solved by:
- Look for the '/images' directory and take the bigger image (by filename perhaps) and be done with it.
- Create some logic to get the resolution of the monitor where the wallpaper is displaying and use the image with that size if the wallpaper provides it, and fallback to the bigger one if doesn't.

Plasma itself has a logic to do the second but doesn't seem to store the selected image anywhere (as far as I know), which I find weird as it used to do it `5.25`, but even there it did return a directory sometimes. 


Now, with the day/night wallpapers, Plasma decides which (normal/dark) wallpaper to use according to the color scheme selected (dark/light). 
The script *should* take that into account too and apply the normal or dark variant of the scheme (from the config or flags) matching the wallpaper, which will change automatically after the new scheme is applied, replicating what Plasma does with the automatic color scheme that gets the colors from the wallpaper.

**Comment by bentuhana** [comment]
Fixed with #98, but im not sure if it should select the higher quality picture? Since its only generating a color scheme, using lowest resolution should not be a problem. Using the largest picture will slow down the read and manipulation processes, which is not good for old hardware.

**Comment by luisbocanegra** [comment]
Hmm, I used the largest because it should (at least most of the time) the one that is the least cropped, and mistook the max/min function as length based so it won't return the smallest/largest either 😆. And while the resizing doesn't take that long in my computer I agree taking the largest picture is not the best approach.
